src/Shared/RedisSharedConnection.cs

```diff
@@ -103,6 +105,10 @@ public void ForceReconnect()
 
                     if ((elapsedSinceFirstError >= ReconnectErrorThreshold) && (elapsedSinceMostRecentError <= ReconnectErrorThreshold))
                     {
+                        logger.Info($"ForceReconnect: now: {utcNow.ToString()}");
+                        logger.Info($"ForceReconnect: elapsedSinceLastReconnect: {elapsedSinceLastReconnect.ToString()}, ReconnectFrequency: {ReconnectFrequency.ToString()}");
+                        logger.Info($"ForceReconnect: elapsedSinceFirstError: {elapsedSinceFirstError.ToString()}, elapsedSinceMostRecentError: {elapsedSinceMostRecentError.ToString()}, ReconnectErrorThreshold: {ReconnectErrorThreshold.ToString()}");
+
                         LogUtility.LogInfo($"ForceReconnect: now: {utcNow.ToString()}");
```

Comment:
Where is LogUtility logging to?

Reply:
We don't use that. To enable logging via LogUtility, we need to specify the class name in the configuration. This was one of the approaches we discussed.